letsencrypt-auto-source/pieces/bootstrappers/rpm_python3.sh

```diff
@@ -0,0 +1,25 @@
+# If new packages are installed by BootstrapRpmPython3 below, this version
+# number must be increased.
+BOOTSTRAP_RPM_PYTHON3_VERSION=1
+
+BootstrapRpmPython3() {
+  # Tested with:
+  #   - CentOS 6 (EPEL must be installed manually)
```

Comment:
I just copied this comment from the previous script. Is the comment made untrue by something changed in this PR, or was it just never updated previously? And this code runs at the same point as previously, so why would it be true for this PR but not true previously?

Reply:
I forgot to update it previously apparently. See #3259.

If you don't fix my mistake, we should at least not duplicate it here.

Reply:
How are we now requiring the user to manually install EPEL here in a way that we weren't before? This script uses the same logic at the same points in code execution, with respect to when we run `$tool list python34`.

Reply:
Because the result of `$tool list python34` depends on whether or not EPEL is installed.

`$tool list <package>` searches for a package with that name using the installed repositories. `$tool list python34` will fail until EPEL is installed because `python34` isn't available in the standard repositories, only in EPEL. If this bootstrap function is run on a CentOS 6 system without EPEL already installed, the `if $tool list python34` branch will not be taken.

Reply:
For more info here, the code that installs EPEL if necessary is the block that starts with `if ! $tool list *virtualenv >/dev/null 2>&1; then`.

Reply:
I feel like we're having a miscommunication here, so maybe we should talk in person tomorrow.

I understand why EPEL is needed to run this code, that's not what I'm asking. But don't we hit the code that installs EPEL before we get here? The way the code was before I ran this, we needed EPEL to check about python 2 packages, now we still need it to check about python 3 packages. In both cases, the code runs after we've installed EPEL for them. What am I missing that I've changed here to make it so we need EPEL earlier than we previously needed it?

Reply:
Ah now I see. It's because we were previously running the enable EPEL code before checking for python packages, and now we're running it after checking for python packages because my refactoring changed the order.